rxtools/source/lib/draw.c

```diff
@@ -7,165 +7,155 @@
 #include "draw.h"
 #include "filepack.h"
 
-int current_y = 1;
+u32 current_y = 1;
 
-u8* tmpscreen = (u8*)0x26000000;
+u8 *tmpscreen = (u8*)0x26000000;
 
-void ClearScreen(unsigned char *screen, int color)
+void ClearScreen(u8 *screen, u32 color)
 {
-    u32 i = SCREEN_SIZE/4;  //Surely this is an interger.
-    u32* tmpscr = (u32*)screen; //To avoid using array index which would decrease speed.
-    //Prepared 3 u32, that includes 4 24-bits color, cached. 4x(BGR)
-    u32 color0 = (color) | (color << 24),
-        color1 = (color << 16) | (color >> 8),
-        color2 = (color >> 16) | (color << 8);
-    while (i--) {
-        *(tmpscr++) = color0;
-        *(tmpscr++) = color1;
-        *(tmpscr++) = color2;
-    }
+	u32 i = SCREEN_SIZE/sizeof(u32);  //Surely this is an interger.
```

Comment:
I'm sorry but we may need to change it to `SCREEN_SIZE/BYTES_PER_PIXEL` cause it should means WIDTH*HEIGHT. Or else it would forget some pixels.